plugins/org.wso2.developerstudio.eclipse.artifact.localentry/src/org/wso2/developerstudio/eclipse/artifact/localentry/validators/LocalEntryProjectFieldController.java

```diff
@@ -166,7 +166,7 @@ public boolean isEnableField(String modelProperty, ProjectDataModel model) {
 	
 	public boolean isReadOnlyField(String modelProperty, ProjectDataModel model) {
 		boolean readOnlyField = super.isReadOnlyField(modelProperty, model);
```

Comment:
Shall we format this method? line 168 and 172 should have same indent from the left border.

Reply:
Fixed in https://github.com/wso2/devstudio-tooling-esb/pull/272